wolfcrypt/benchmark/benchmark.c

```diff
@@ -2842,6 +2842,11 @@ int benchmark_init(void)
 
     benchmark_static_init(0);
 
+#ifdef WOLFSSL_STM32_CUBEMX
+    /* enable the peripheral clock */
+    __HAL_RCC_CRYP_CLK_ENABLE();
```

Comment:
What about having this in stm32.c in `wc_Stm32_Aes_Init` wrapped with a macro that is off by default? If you need add a `wc_Stm32_Aes_Cleanup`.